views/includes/page-templates/trust-individual-beneficial-owner.html

```diff
@@ -58,18 +58,23 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageTitle }}</h1>
 
       {% include "includes/inputs/text-input.html" %}
 
-      {% set fieldParam = {
-        name: 'dateOfBirth',
-        value: {
-          'Day': formData.dateOfBirthDay,
-          'Month': formData.dateOfBirthMonth,
-          'Year': formData.dateOfBirthYear
-        },
-        label: 'What is their date of birth?',
-        error: errors.dateOfBirth if errors
-      } %}
-
-      {% include "includes/inputs/date-input.html" %}
+      {% if uneditableDOB %}
```

Comment:
Would be better in a template to include than inline here

Reply:
No, just wrapped existing HTML with the if statement is the only change here.